### PR TITLE
Fresh

### PR DESCRIPTION
**Pull request by juliaputko**

### CODE REVIEW

**Review by jklymak — COMMENTED** [review]
This is looking good.  A few minor comments.

File: datapage.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      </p>
+    </td>
+  </tr>
+ 
+</table>
+
+
+<h2>Data Description</h2>
+
+The different types of data files that are available are named as follows:
```
Comment (hvdosser) [inline datapage.html:36]: Can we change this sentence to "As of 2022, the different types of data files that are available are named as follows: " ?
Then I'll put it back after I catch up on updating the old directories to match the new naming conventions.

File: datapage.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+</ul>
+
+
+
+
+    <h2>Data Processing  </h2>
+
+    <table style="width:100%">
+      <tr>
+        <td><a href="test.pdf" target="_blank">Open PDF</a></td>
```
Comment (hvdosser) [inline datapage.html:52]: Let's remove the blank PDFs for now (just comment out if possible), so they don't look like broken links.

**Review by hvdosser — CHANGES_REQUESTED** [review]
Julia, can you take a look at all of my and Jody's comments, and either reply or make changes as is appropriate? The main concern is whether some of the git pages development code accidentally made it into the pull request. .

File: _includes/head.html [hunk 3]
```diff
@@ -18,17 +18,21 @@
 
     <link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
 
-    <!-- Bootstrap Core CSS -->
-    <link rel="stylesheet" href="{{ "/css/bootstrap.css" | prepend: site.baseurl | prepend: site.url }}">
+      <!-- Bootstrap Core CSS -->
+    <!--link rel="stylesheet" href="{{ "/css/bootstrap.css" | prepend: site.baseurl | prepend: site.url }}">-->
```
[the remark this answers is not in the document]
Comment (hvdosser) [reply _includes/head.html:22]: It's been re-added now.

File: index.html [hunk 4]
```diff
@@ -15,39 +15,94 @@
...
+
+<p>The Canadian-Pacific Robotic Ocean Observing Facility was initiated with CFI/BCKDF funding to build ocean observing
+   capacity off the British Columbia coast. It operates a fleet of autonomous gliders, floats, and moorings providing 
+   ocean scientists with long-term monitoring of the ocean at the small scales important to resolve upper ocean physical 
+   and biological properties.</p>
 
 <h3> Deployments </h3>
 
-C-PROOF glider deployments are under way!  Click on map to see list of deployments and access data:
-<a href="deployments/">
```
Comment (jklymak) [inline index.html:25]: OK< I see - we should add this link back in around the map please...

**Comment by juliaputko** [comment]
Hi Jody, 


_includes/head.html <https://github.com/c-proof/cproofwebsite/pull/6#discussion_r838142525>:
I removed site-url because I was having issues loading the css files during my testing, however I tested again just now, and it does not seem to be an issue, so I can revert it. 

_layouts/data_jp.html <https://github.com/c-proof/cproofwebsite/pull/6#discussion_r838142751>
I use this file as a layout for the data.md page. To make the navigation bar, jekyll looks for the word “project” in the front-matter of layouts, so I made a separate layout for this page because I did not want it to appear in the top navigation bar. 

about.m <https://github.com/c-proof/cproofwebsite/pull/6#discussion_r838143293>d :
I will review the website you sent, and also talk about it with Hayley. 

data.md <https://github.com/c-proof/cproofwebsite/pull/6#discussion_r838144049>
This page is accessible through the ‘wget data’ button on the deployments page. 
As for wgetting the data, this is also something decided by Hayley, so I will need to bring this up with her as well. 

As for changing to raw html in markdown, I made css additions, and as to not clutter the index file with my css, I have added all the css to the bootstrap.css file - however for organizational purposes, and so as to not have to rename everything I added a class so that the css I added could be categorized under that class. (eg. .platforms .p { #css changes } )
> Is there a reason you changed to raw html in a markdown file? That usually shouldn't be needed…"
> 
> <body class="platforms"
> 
> 

Best, 
Julia Putko

**Comment by juliaputko** [comment]
Hi Hayley, 

The files added to .gitignore were added because unless I added them, my installation of jekyll site related things would result in 10,000+ git changes among those related files. 

I changed the name of the class in members.html. jptest was a name that I had chosen early on, and never changed. It is changed to people_ now. 

cproof-deployments_argo.geojson is now cproof-deployments_all.geojson 

I changed that one line in the description on the data page. 

The blank pdfs are commented out. 


I will make a new commit with all those changes, it should just append itself to the last pull request. 


Best, 
Julia Putko

**Pull request merged**